src/Providers/AbstractServiceProvider.php

```diff
@@ -86,7 +88,9 @@ private function _getDriver($driverName)
             $driverClass = sprintf('KolayIK\\Auth\\Drivers\\%s', $driverName);
```

Comment:
Please use an associated array to be easier to rename or move classes in future.